js/interface/map-panel.js

```diff
@@ -22,7 +22,50 @@ Fliplet.InteractiveMap.component('map-panel', {
       default: true
     }
   },
+  data() {
+    return {
+      updateDebounced: _.debounce(this.updateDataSource, 1000),
+      appId: _.keys(__widgetData)[0],
+      dataSourceId: undefined,
+      entries: undefined,
+      columns: undefined,
+      dataSourceConnection: undefined,
+      decidedToKeepMarkers: false,
+      imageWidth: undefined,
+      imageHeight: undefined,
+      oldMapName: ''
+    }
+  },
   methods: {
+    saveToDataSource() {
+      this.dataSourceConnection.commit(this.entries, this.columns)
+      this.oldMapName = this.name
+      Fliplet.Studio.emit('reload-widget-instance', this.appId)
+    },
+    getMapName() {
+      this.oldMapName = this.name
+    },
+    updateDataSource() {
+      this.dataSourceId = __widgetData[this.appId].data.markersDataSourceId
```

Comment:
see other comment, this should not be used as it is as it won't work

Reply:
Fixed